.github/scripts/update-manifests-tags.sh

```diff
@@ -14,4 +14,8 @@ env | while IFS="=" read varname value; do
     component=${varname#${prefix}}
     component=${component//_/-}
     update_tags $(basename $component) $value
-done
\ No newline at end of file
+done
+
```

Comment:
do we have a problem to make this a generic function?

is this the reason  why you thought it is a hacky to get Trusty use a different "org"? 
things for example should be able to replaced by input from "Tracker" to 
`    ["component_name"]="org:repo_name:repo_branch:path"`
in file, right? 
as long as the key "component_name" can be found in both get_all_manifests.sh and the Track input.

or do i misunderstand something here?

Reply:
my concern was that, this is a very rare occurrence where we use a different org to get the manifests. I can update the workflow, but it will be change which will be very less used.